release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2133,6 +2156,10 @@ To make the required change, follow the instructions in link:https://access.redh
 +
 (link:https://issues.redhat.com/browse/OCPBUGS-15997[*OCPBUGS-15997*])
 
+* For clusters that run on {rh-openstack} 17.1 and use network function virtualization (NFV), a known issue in {rh-openstack} prevents succesful cluster deployment. There is no workaround for this issue. A hotfix that addresses the problem will be released soon.
```

Comment:
@MaysaMacedo Not sure what to put here for that OSP bug--is it likely that the hotfix will be released before OCP 4.14 GA?

Reply:
@maxwelldb It's unlikely that the fix will be released before OCP GA. Perhaps the following could be mentioned instead: "While there is not fix released, customers can request a hotfix".

@eurijon what do you think?

Reply:
```suggestion
* For clusters that run on {rh-openstack} 17.1 and use network function virtualization (NFV), a known issue in {rh-openstack} prevents succesful cluster deployment. There is no workaround for this issue. Contact Red Hat Support to request a hotfix.
```

Reply:
The change looks good, the OSP fix won't be available before 4.14 GA.
cc @zivgreenberg